openlibrary/plugins/openlibrary/js/signup.js

```diff
@@ -168,19 +185,40 @@ export function initSignupForm() {
         }
     }
 
-    // Validates inputs already marked as invalid on value change
-    $('form[name=signup] input').on('input', debounce(function(){
+    const $nonCheckboxInputs = $('form[name=signup] input:not([type="checkbox"])')
```

Comment:
Selector has been updated to avoid passing checkboxes to the `validateInput()` function, which throws an error when a given input has no associated validation function.

The existing `ia_newsletter` checkbox has no validation function, and throws errors when blurred in production today.  The new `pd_request` checkbox and `pd_program` select element are only validated on page load and submit button click, so they do not need `blur` or `input` handlers.